pkg/node/bootstrap.go

```diff
@@ -214,7 +219,12 @@ func bootstrapNode(
 		eventsJSON        []byte
 	)
 
+	logger = passedInLogger
+
 	for i := 0; i < getSnapshotRetries; i++ {
+		if err != nil {
+			time.Sleep(retryWait)
```

Comment:
why do you think that retrying would make any difference? the snapshot is 1mb... we should not even run into any time-settlements issues afaik but i might be wrong

Reply:
So I am guessing the difference could be that we have some more peers connected?